### ISSUE TITLE
Cannot initialize Betterpushback

### ISSUE THREAD

**Issue by REJHEB** [skiselkov/BetterPushbackC#237]
I can load Betterpushback but X-Plane cannot enabled it.
I run X-Plane 10 in Windows 10
Heree is my log.txt
[Log.txt](https://github.com/skiselkov/BetterPushbackC/files/2401900/Log.txt)
I hope someone will help me resolve that issue
Thanks in advance
RH

**Comment by meltinglava** [comment]
Your log states:
`2018-09-20 09:49:07 BetterPushback[airportdb.c:1259]: navdata error: your Airports.txt is missing or unreadable. Please correct this and recreate the cache.`
Can you check if you have any files called this.
I have:
```
.\Custom Data\UFMC\Airports.txt
.\Custom Data\GNS430\navdata\Airports.txt
```

**Comment by REJHEB** [comment]
Yes I have this file in  X-plane 10\Custom Data\UFMC\Airports.txt.
Here is the first few lines:

X,1707,22JUN19JUL/17,1706,25MAY21JUN/17

A,01ID,LAVA HOT SPRINGS,42.608250,-112.032461,5268,18000,18000,2800,0
R,14,143,2894,100,0,0.000,0,42.611000,-112.034867,5268,3.00,50,2,0
R,32,323,2894,100,0,0.000,0,42.603733,-112.030533,5268,3.00,50,2,0

A,01NK,GRANVILLE,43.425000,-73.261944,421,18000,18000,2500,0
R,16,160,2500,50,0,0.000,0,43.425000,-73.261944,421,3.00,50,1,0
R,34,340,2500,50,0,0.000,0,43.425000,-73.261944,421,3.00,50,1,0

Thank you for answering  my issue,
Réjean

**Comment by meltinglava** [comment]
Might be an X-plane 10 problem. I cant tell you more.
